xbmc/utils/CharsetConverter.cpp

```diff
@@ -496,7 +501,7 @@ bool CCharsetConverter::CInnerConverter::logicalToVisualBiDi(const std::u32strin
       lineEnd = srcLen;
     else
       lineEnd++; // include '\n'
-
+    
```

Comment:
oh yeah those snuck in because I tried som VS addon to remove whitespace on save, in one of the earlier commits so too lazy to revert the changes. No harm done other than some noise in the diff